src/amber/cli/helpers/process_runner.cr

```diff
@@ -1,114 +1,233 @@
+require "http"
 require "./helpers"
+require "../config"
+require "./file_watcher"
+require "../../exceptions/exception_page"
 
-module Sentry
-  class ProcessRunner
-    property processes = [] of Process
-    property process_name : String
-    property files = [] of String
-    @logger : Amber::Environment::Logger
-    FILE_TIMESTAMPS = {} of String => String
-
-    def initialize(
-      @process_name : String,
-      @build_command : String,
-      @run_command : String,
-      @build_args : Array(String) = [] of String,
-      @run_args : Array(String) = [] of String,
-      files = [] of String,
-      @logger = Amber::CLI.logger
-    )
-      @files = files
-      @npm_process = false
-      @app_running = false
+module Amber::CLI::Helpers
+  include Environment
+
+  struct ProcessRunner
+    PROCESSES = [] of {Process, String}
+
+    @host : String
+    @port : Int32
+    @file_watcher : FileWatcher
+
+    def initialize
+      @watch_running = false
+      @wait_build = Channel(Bool).new
+      @server_files_changed = false
+      @notify_counter = 0
+      @notify_counter_channel = Channel(Int32).new
+      @notify_channel = Channel(Nil).new
+      @host = Helpers.settings.host
```

Comment:
How come we are getting settings information from a Helper module and not Amber::CLI environment settings?

Reply:
Oh, this is using `include Enviroment`, I know this is a bad practice, because even I was confused about this when I tried to figure out where `Helpers.settings` is defined, So, I gonna use `Amber::CLI` :+1: